src/Control/SimpleSearchField.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+namespace ipl\Web\Control;
+
+use ipl\Html\Attributes;
+use ipl\Html\Form;
+use ipl\Html\FormElement\InputElement;
+use ipl\Html\FormElement\SubmitElement;
+use ipl\Html\HtmlElement;
+use ipl\Web\Url;
+
+/**
+ * Form for simple value based search
+ */
+abstract class SimpleSearchField extends Form
```

Comment:
Why is this abstract? It doesn't define any abstract methods. I also see no reason why it should not be used directly.